src/ProjectTemplates/Web.ItemTemplates/content/RazorView/.template.config/localize/templatestrings.en.json

```diff
@@ -1,6 +1,7 @@
 {
   "author": "Microsoft",
   "name": "Razor View",
-  "description": "A Razor view without a page model",
-  "postActions/openInEditor/description": "Opens Index.cshtml in the editor"
-}
+  "description": "An empty Razor view",
+  "_description.comment": "Reviewers: Could also be 'A Razor view without a page model'. And should View be capitalized?",
```

Comment:
Let's capitalize the `View` but leave the text as is for now.

Reply:
done